client/package.json

```diff
@@ -57,15 +56,14 @@
     "postpackage": "rimraf ./tmp"
   },
   "devDependencies": {
-    "@sentry/types": "6.19.1",
     "@types/prettier": "2.6.0",
     "@types/vscode": "^1.90",
     "rimraf": "3.0.2",
     "prettier": "2.5.1"
   },
   "dependencies": {
     "@nomicfoundation/solidity-language-server": "0.8.20",
-    "@sentry/node": "6.19.1",
```

Comment:
Should the client change to be `@sentry/browser`?
This is based on https://docs.sentry.io/platforms/javascript/best-practices/shared-environments/

Reply:
Since our extension runs on a node js environment, I'm confident the @sentry/node package is the more apropiate one. Checking the documentation page, it may be confusing because sometimes it refers to browser extensions themselves, but vscode extensions are treated as a separate concept. Although if there is any potential advantage on the `@sentry/browser` package that we may be interested in, I can dive deeper into it

Reply:
Ah okay, so I have misunderstood.
It is not that we should swap to the reduced browser client, but that we should use the initialization techniques (which they demonstrate with the browser client).

Reply:
I am happy to go with the node version and determine from Sentry whether it achieves our goals. Specifically:

* No uncaught exceptions from other extensions
* ideally no phantom extensions in our sentry reports.

Reply:
When working on this I've created a dummy extension alongside that periodically crashed to ensure errors from that extension wouldn't get in the way. For process unhandled errors, there's a special check that would report them only if our extension is on the stack trace (line 58 on SentryClientTelemetry)